pkg/state/block_differ_test.go

```diff
@@ -38,7 +38,7 @@ func genBlocks(t *testing.T, to *blockDifferTestObjects) (*proto.Block, *proto.B
 	txs := proto.Transactions{createTransferWithSig(t)}
 	randSig := genRandBlockIds(t, 1)[0]
 	genSig := crypto.MustBytesFromBase58(defaultGenSig)
-	parent, err := proto.CreateBlock(txs, 1565694219644, randSig, testGlobal.matcherInfo.pk, proto.NxtConsensus{BaseTarget: 65, GenSignature: genSig}, proto.NgBlockVersion, nil, -1)
```

Comment:
`TestNetScheme` -> `MainNetScheme`. This block is not Protobuf block, so scheme does not really affect anything, but it is then Signed() using `MainNetScheme`, so it should be the same for clarity.